designs/lily/src/back.mjs

```diff
@@ -0,0 +1,567 @@
+      // Help construct cross seam
+      paths.hint = new Path()
+        .move(points.crossSeamCurveStart)
+        .line(points.crossSeamCurveMax)
+        .line(points.fork)
+        .attr('class', 'note lashed')
+      macro('hd', {
+        from: points.floorIn,
+        to: points.floorOut,
+        y: points.floorIn.y - 30,
+      })
+      macro('hd', {
+        from: points.floorIn,
+        to: points.floor,
+        y: points.floorIn.y - 15,
+      })
+      macro('hd', {
+        from: points.floor,
+        to: points.floorOut,
+        y: points.floorIn.y - 15,
+      })
+      macro('vd', {
+        from: points.floorOut,
+        to: points.styleWaistOutLily,
+        x:
+          (points.seatOut.x > points.styleWaistOutLily.x ? points.seatOut.x : points.styleWaistOutLily.x) +
+          sa +
+          15,
+      })
+      macro('vd', {
+        from: points.floorIn,
+        to: points.fork,
+        x: points.fork.x - sa - 15,
+      })
+      macro('vd', {
+        from: points.fork,
+        to: points.styleWaistInLily,
+        x: points.fork.x - sa - 15,
+      })
+      macro('vd', {
+        from: points.floorIn,
+        to: points.styleWaistInLily,
+        x: points.fork.x - sa - 30,
+      })
+      macro('vd', {
+        from: points.crossSeamCurveStart,
+        to: points.styleWaistInLily,
+        x: points.crossSeamCurveStart.x - sa - 15,
+      })
+      macro('hd', {
+        from: points.styleWaistInLily,
+        to: points.grainlineTop,
+        y: points.styleWaistInLily.y - sa - 15,
+      })
+      macro('hd', {
+        from: points.crossSeamCurveStart,
+        to: points.grainlineTop,
+        y: points.styleWaistInLily.y - sa - 30,
+      })
+      macro('hd', {
+        from: points.crossSeamCurveMax,
+        to: points.grainlineTop,
+        y: points.styleWaistInLily.y - sa - 45,
+      })
+      macro('hd', {
+        from: points.fork,
+        to: points.grainlineTop,
+        y: points.styleWaistInLily.y - sa - 60,
+      })
+      macro('hd', {
+        from: points.grainlineTop,
+        to: points.styleWaistOutLily,
+        y: points.styleWaistInLily.y - sa - 15,
+      })
+      if (points.seatOut.x > points.styleWaistOutLily.x) {
+        macro('hd', {
+          from: points.grainlineTop,
+          to: points.seatOut,
+          y: points.styleWaistInLily.y - sa - 30,
+        })
+      }
+    }
+  }
+
+  return part
+}
+
+export const back = {
+  name: 'lily.back',
+  from: titanBack,
+  //after: titanFront,
+  measurements: [
+    'ankle',
+    'heel', // secondary measurement, used instead of ankle
+  ],
+  options: {
+    fitGuides: {bool: true, menu: undefined},
+    fitKnee: {bool: true, menu: undefined},
+    legBalance: 0.5, // between back and front parts
+    waistBalance: 0.5,
+    waistHeight: { ...titanBack.options.waistHeight, pct: 50 }, // halfway between waist and hips
+    fabricStretch: { pct: 40, min: 0, max: 50, menu: 'fit' },
```

Comment:
fabricStretch seems to affect only the width of the lower leg. The waistband and seat/upper leg stay the same regardless of the amount of fabricStretch. This seems to be intentional, but I am not sure.

Reply:
Yes, the other widths are controlled via their respective ease values. Ideally, fabricStretch would control those values, but I haven't yet found a way to do so.